fusedrug/data/interaction/drug_target/loaders/indexed_text_PPI_table_loader.py

```diff
@@ -0,0 +1,50 @@
+from typing import Optional
+from fuse.utils import NDict
+from fuse.data import OpBase, get_sample_id
+from fusedrug.utils.file_formats import IndexedTextTable
+
+
+class IndexedTextPPITableLoader(OpBase):
+    """
+    Loads an entry from an smi file (large files are supported!).
+    """
+
+    def __init__(
+        self,
+        table_file_loc: Optional[str] = None,
```

Comment:
Why is it optional?

Reply:
remnant from someone else's code. Changed to str